styles.css

```diff
@@ -162,293 +132,559 @@ a:active {
+    margin-top: 10rem;
+  }
+
+  .about {
+    margin: 60px 0;
+    font-size: 14px;
+    width: 70%;
+  }
+
+  .collab-btn,
+  .cv-btn {
+    align-items: center;
+    width: 254px;
+    height: 70px;
+  }
+
+  /* projects section  */
+
+  .firstproject {
+    margin-left: 100px;
+    display: flex;
+    flex-direction: row;
+    margin-top: 200px;
+  }
+
+  .firstproject .work_cards {
+    width: 70%;
+    background-image: url(img/image_dots.svg);
+    background-position: top left;
+    margin: auto;
+    background-repeat: no-repeat;
+  }
+
+  .firstproject .workdetails {
+    height: 250px;
+    width: 81.2%;
+    margin-top: -250px;
+    margin-left: auto;
+  }
+
+  .workdetails button {
+    width: 12vw;
+    font-size: 14px;
+    background: white;
+    color: #1a2236;
+  }
+
+  .firstproject .work_image {
+    margin-left: auto;
+    width: 83%;
+  }
+
+  .firstproject_name {
+    background-repeat: no-repeat;
+    background-position: bottom right;
+    margin-left: -11rem;
+    width: 40%;
+    height: auto;
+    margin-top: -50px;
+  }
+
+  .firstproject h2 {
+    transform: rotate(-90deg);
+  }
+
+  .sec_third {
+    width: 98vw;
+    display: flex;
+    flex-direction: row;
+  }
+
+  #card2 {
+    height: 350px;
+    width: 65%;
+    background-image: url(img/image-geometry_circle\ and\ dot.svg);
+    background-repeat: no-repeat;
+    background-position: right;
+  }
+
+  #card2 img {
+    margin-top: 80px;
+    width: 47.5%;
+  }
+
+  #card3 {
+    margin-left: auto;
+    width: 35%;
+  }
+
+  #card3 img {
+    margin-left: auto;
+    width: 80%;
+  }
+
+  #card3 .workdetails {
+    width: 98.9%;
+    height: 195px;
+    margin-top: -250px;
+  }
+
+  .four_five {
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: space-between;
+    width: 100%;
+    margin-top: 0;
+    height: 55.2vh;
+  }
+
+  #card4 {
+    margin-top: -20px;
+    width: 50%;
+    background-image: url(img/image-geometry_merge.svg);
+    background-repeat: no-repeat;
+    background-position: bottom right;
+  }
+
+  #card4 img {
+    width: 84.9%;
+    margin-top: -74px;
+    margin-left: auto;
+  }
+
+  #card5 {
+    width: 69%;
+  }
+
+  #card5 img {
+    height: auto;
+  }
+
+  .lastcard {
+    width: 87.5vw;
+    display: flex;
+  }
+
+  #card2 .workdetails {
+    display: none;
+  }
+
+  #card4 .workdetails {
+    display: none;
+  }
+
+  #card5 .workdetails {
+    display: none;
+  }
+
+  #card6 .workdetails {
+    display: none;
+  }
+
+  .lastbg {
+    width: 95%;
+    background:
+      url(img/image-geometry_5.svg) top left no-repeat,
+      url(img/image-geometry_yellow.svg) top right no-repeat;
+  }
+
+  #card6 {
+    width: 50%;
+  }
+
+  .midimage {
+    display: grid;
+    height: 47vh;
+    background-size: 100% 100%;
+  }
+
+  #about-section {
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: space-between;
+    margin-right: 100px;
+    margin-top: 210px;
+    margin-bottom: 150px;
+  }
```

Comment:
- [ ] Please kindly apply margin to the` about me section` so as to resemble the original template. 👍 

## Your Design

![b10](https://user-images.githubusercontent.com/103386183/199995154-003b1473-d399-4747-80db-20cbcedbf0d2.png)

## Figma Design

![b8](https://user-images.githubusercontent.com/103386183/199995289-2a6c53b0-a75d-4b2d-a3f7-9360522f3df3.png)